be/src/pipeline/pipeline_task.cpp

```diff
@@ -315,7 +315,8 @@ bool PipelineTask::should_revoke_memory(RuntimeState* state, int64_t revocable_m
     wg->check_mem_used(&is_wg_mem_low_water_mark, &is_wg_mem_high_water_mark);
     if (is_wg_mem_high_water_mark) {
         if (revocable_mem_bytes > min_revocable_mem_bytes) {
-            LOG_EVERY_N(INFO, 10) << "revoke memory, hight water mark";
+            LOG_EVERY_T(INFO, 1) << "query " << print_id(state->query_id())
```

Comment:
Not print this log, it should be a debug log. 
This method is called by pipeline engine during every loop.
If there is many queries, the log is too often.